woocommerce-services.php

```diff
@@ -843,7 +843,7 @@ public function rest_api_init() {
 			$this->set_rest_self_help_controller( $rest_self_help_controller );
 			$rest_self_help_controller->register_routes();
 
-			require_once __DIR__ . 'classes/class-wc-rest-connect-service-data-refresh-controller.php' ) );
+			require_once __DIR__ . 'classes/class-wc-rest-connect-service-data-refresh-controller.php';
```

Comment:
There seems to be a leading slash missing here:

```suggestion
			require_once __DIR__ . '/classes/class-wc-rest-connect-service-data-refresh-controller.php';
```

Reply:
🦅 👀 
I really messed up that merge conglict, huh? :D

Reply:
Good catch! Thanks!! Fixed.